<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BezClass</class>
 <widget class="QMainWindow" name="BezClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BezClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineFirstX"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineSecondX"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineThirdX"/>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineFourX"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineFirstY"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineSecondY"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineThirdY"/>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineFourY"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Кривая по 4-ём точкам</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
